packages/api/src/feature_flag/service/create.ts

```diff
@@ -0,0 +1,50 @@
+import { logger } from "@stela/logger";
+import createError from "http-errors";
+import type { CreateFeatureFlagRequest, FeatureFlagRow } from "../models";
+import { db } from "../../database";
+
+export const createFeatureFlag = async (
+  featureFlagData: CreateFeatureFlagRequest
+): Promise<FeatureFlagRow> => {
+  const featureFlagsWithName = async (name: string): Promise<boolean> => {
+    const result = await db
+      .sql<FeatureFlagRow>("feature_flag.queries.get_feature_flag", {
+        name,
+      })
+      .catch((err) => {
+        logger.error(err);
+        throw new createError.InternalServerError(
+          "Failed to retrieve feature flags"
+        );
+      });
+
+    return result.rows.length > 0;
+  };
+
+  if (await featureFlagsWithName(featureFlagData.name)) {
+    throw new createError.BadRequest("Feature flag already exists");
+  }
+
+  const result = await db
+    .sql<FeatureFlagRow>("feature_flag.queries.create_feature_flag", {
+      name: featureFlagData.name,
+      description: featureFlagData.description,
+      globally_enabled: true,
```

Comment:
`globally_enabled` should always be `false` when a feature flag is created, because at the moment of creation the feature flag should be turned off

Reply:
I see, ok, done. I saw it as true in the API docs and thought we want that as the default

Reply:
Ah, yes, it appears to be the default behavior of redocly that the example value of any boolean field is `true`, which is misleading in this case. It's probably worth adding a description to the `globallyEnabled` fields in the docs noting that it always starts out as `false.